README.md

```diff
@@ -25,15 +25,16 @@ $ cd onlineweb4
 
 To fire up the dev environment, you should use docker-compose.
 
-Install it by running `pip install docker-compose`.
+First install Docker and Docker-compose 
 
-There exists a `Makefile` in the project root directory. This simplifies interaction with docker and docker-compose.
+Run navigate to project root folder and run 'docker-compose up' 
 
-Simply run `make` to build and start onlineweb4, and run `make stop` to stop it.
+Run bach in the container with 'docker exec -i -t onlineweb4_django_1 bash'
+rename settings-file in onlineweb4/settings from example-local.py to local.py 
```

Comment:
This is not needed. We can override settings using environment variables instead, but creating a local.py is still supported.